tests/suites/test_suite_pk.function

```diff
@@ -417,26 +417,30 @@ exit:
 void pk_rsa_overflow( )
 {
     mbedtls_pk_context pk;
-    size_t hash_len = SIZE_MAX;
+    size_t hash_len = SIZE_MAX, sig_len = SIZE_MAX;
+    unsigned char hash[50], sig[1000];
```

Comment:
Please use a smaller buffer for `sig`. This is a preexisting issue in other test cases as well, the large sizes cause stack overflows on some platforms, we're fixing them little by little as we modify the code for other reasons.